apps/web/src/services/asset-inventory/collector/shared/collector-data-modal/modules/CollectorDataDuplicationInner.vue

```diff
@@ -2,7 +2,7 @@
     <div class="collector-data-duplication-inner">
         <span>{{ $t('INVENTORY.COLLECTOR.MAIN.COLLECT_DATA_MODAL.DUPLICATION_DESCRIPTION') }}</span>
         <p-definition-table :fields="definitionFields"
-                            :data="tableState.definitionData"
+                            :data="state"
```

Comment:
issue?: I think `state.status` should be used?

Reply:
Because `duration` also needs to be used, the encompassing state should be passed!